engine/input/src/input.cpp

```diff
@@ -559,31 +559,9 @@ namespace dmInput
         action->m_HasGamepadPacket = 0;
     }
 
-    struct UpdateActionContext
```

Comment:
No need to store action values here and iterate over all actions and set them. Instead we set the values when we update the relevant actions.